### PR TITLE
Fix comments in @code blocks in fe headers

### PR DESCRIPTION
**Pull request by masterleinad**
Relates to #6814.

### CODE REVIEW

File: include/deal.II/fe/fe_enriched.h [hunk 1]
```diff
@@ -251,15 +251,15 @@ class FE_Enriched : public FiniteElement<dim, spacedim>
...
-   * Function<dim> * {return &fe_1_function1;},
-   *   [=] (const typename Triangulation<dim>::cell_iterator &) -> const
-   * Function<dim> * {return &fe_1_function2;}},
-   *  {[=] (const typename Triangulation<dim>::cell_iterator &) -> const
-   * Function<dim> * {return &fe_2_function;}}});
+   * FE_Enriched<dim> fe(
+   *   &fe_base,
+   *   {&fe_1, &fe_2},
+   *   {{[=] (const typename Triangulation<dim>::cell_iterator &)
+   *       -> const Function<dim> * {return &fe_1_function1;},
```
Comment (bangerth) [inline include/deal.II/fe/fe_enriched.h]: Can you break the body of the lambda onto a line of its own? Or does reflow-comments destroy that again? 

Same below of course.

Comment (masterleinad) [reply include/deal.II/fe/fe_enriched.h]: ReflowComments will only touch lines which are longer than 80 characters. I can do that change.

Comment (bangerth) [reply include/deal.II/fe/fe_enriched.h]: Thanks. The existing style look too much like we're multiplying `Function<dim>` by the thing in braces :-)

**Pull request merged**